src/main/java/seedu/trippie/Ui.java

```diff
@@ -35,5 +35,9 @@ public String readCommand() {
         return in.nextLine();
     }
 
+    public String getLine() {
```

Comment:
What is the difference between this and line 33?

Reply:
Unlike read command, this line is used without printing any symbols before the user input.